### PR TITLE
Support self serve and classroom quiz answers

### PR DESCRIPTION
**Pull request by kkemple**

### CODE REVIEW

File: modules/CONT-04_Editing-pull-request-files.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ screens:
       id: CONT-04-lab-01
       presenter-script:
         - Let's make the changes requested by our teammates.
+      # this is very specific to classroom, is there a way to reword to be event type agnostic?
```
Comment (kkemple) [inline modules/CONT-04_Editing-pull-request-files.yml:36]: @crichID is there a way to get this a bit more agnostic about the event type?

File: modules/COLL-01_Exploring-a-repository.yml [hunk 2]
```diff
@@ -68,9 +68,13 @@ screens:
               id: COLL-01-qq3-o1
               response: "Good job! @githubteacher opened issue #1."
               correct: true
-            - value: # Null means it matches anything else - please make this the last option
+            - value: /hubyeti/i
```
Comment (crichID) [inline modules/COLL-01_Exploring-a-repository.yml:71]: @kkemple So are we saying that hubyeti or githubteacher may open this issue?

File: modules/COLL-01_Exploring-a-repository.yml [hunk 3]
```diff
@@ -85,9 +89,13 @@ screens:
               id: COLL-01-qq4-o1
               response: "Good job! @crichID was the first person to comment on issue #2."
               correct: true
-            - value: # Null means it matches anything else - please make this the last option
+            - value: /hubyeti/i
```
Comment (crichID) [inline modules/COLL-01_Exploring-a-repository.yml:92]: @kkemple same as above - are we leaving the option for either crichID or hubyeti to comment first?

Comment (kkemple) [inline modules/COLL-01_Exploring-a-repository.yml:92]: We are saying or, if its the class then instructor created the comment. If
it is self serve, hubyeti does it, so should accept either until there is a
better way to separate the two event types.

Comment (crichID) [inline modules/COLL-01_Exploring-a-repository.yml:92]: :+1: got it!

**Comment by crichID** [comment]
OK - with explanation this one looks good! I will go ahead and :shipit:

**Pull request merged**